SDL/LST.cc

```diff
@@ -1,12 +1,27 @@
+#ifdef LST_STANDALONE
 #include "LST.h"
+#else
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+#endif
+
 #include "Event.h"
 #include "Globals.h"
 
 #include "Math/Vector3D.h"
 using XYZVector = ROOT::Math::XYZVector;
 
 namespace {
-  std::string trackLooperDir() { return getenv("LST_BASE"); }
+  std::string trackLooperDir() {
+    const char* path = getenv("LST_BASE");
+    std::string path_str;
+    if (path != nullptr) {
+      path_str = path;
+    } else {
+      path_str = getenv("CMSSW_BASE");
```

Comment:
```suggestion
      // FIXME: temporary solution, will need to pass a value from `FileInPath`
      // in the `LSTProducer` or a related ES producer
      path_str = getenv("CMSSW_BASE");
```

Reply:
We're not going to be able to put data files in the CMSSW repo, so I think we'll just have to rework this whole thing.

Reply:
Even as a temporary solution, does it make sense to follow this suggestion?
https://mattermost.web.cern.ch/cms-o-and-c/pl/abax445idjyd8ec4p4ji84zahh